packages/react-internal/package.json

```diff
@@ -55,6 +55,7 @@
     "sinon": "^4.1.3"
   },
   "dependencies": {
+    "@fluentui/react-button": "^1.0.0-beta.6",
```

Comment:
If you're doing this it might be good to remove the direct dep from the main react package and add the re-export files here.

Reply:
I'm having a chat with Jon and David on the side that might lead us to not need this here, but if we do I'll update as per your request.